python/tests/share/test_synthesizer.py

```diff
@@ -11,44 +9,342 @@
     share_lib_path = os.path.join(ResTest.createSharePath("lib"))
 
     sys.path.insert(0, share_lib_path)
-    synthesizer_module =  __import__("synthesizer")
+    synthesizer_module = __import__("synthesizer")
     OilSimulator = synthesizer_module.OilSimulator
     sys.path.pop(0)
 
+if sys.version_info[0] == 3 and sys.version_info[1] >= 2:
```

Comment:
You can do `sys.version_info >= (3, 2)` ;)

Reply:
Done!